specification/communication/resource-manager/Microsoft.Communication/stable/2020-08-20/examples/checkNameAvailabilityAvailable.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "subscriptionId": "12345",
+    "api-version": "2020-80-20",
```

Comment:
API version is incorrect (specified 80 instead of 08). Cross check other places too.